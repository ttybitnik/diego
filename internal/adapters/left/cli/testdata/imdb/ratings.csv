Const,Your Rating,Date Rated,Title,URL,Title Type,IMDb Rating,Runtime (mins),Year,Genres,Num Votes,Release Date,Directors
tt0287467,10,2016-07-24,Hable con ella,https://www.imdb.com/title/tt0287467/,movie,7.9,112,2002,"Drama, Mystery, Romance",116629,2002-03-15,Pedro Almod√≥var
tt0058003,10,2018-08-21,Il deserto rosso,https://www.imdb.com/title/tt0058003/,movie,7.5,117,1964,Drama,17257,1964-09-07,Michelangelo Antonioni
tt0118694,10,2021-05-16,Fa yeung nin wah,https://www.imdb.com/title/tt0118694/,movie,8.1,98,2000,"Drama, Romance",163252,2000-05-20,Kar-Wai Wong