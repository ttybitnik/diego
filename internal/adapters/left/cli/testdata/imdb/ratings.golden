- name: Red Desert
  originalname: Il deserto rosso
  directors: Michelangelo Antonioni
  year: "1964"
  yourrating: "2018-08-21"
  imdbrating: "7.4"
  url: https://www.imdb.com/title/tt0058003
- name: Talk to Her
  originalname: Hable con ella
  directors: Pedro Almod√≥var
  year: "2002"
  yourrating: "2016-07-24"
  imdbrating: "7.9"
  url: https://www.imdb.com/title/tt0287467
- name: In the Mood for Love
  originalname: Fa yeung nin wah
  directors: Kar-Wai Wong
  year: "2000"
  yourrating: "2021-05-16"
  imdbrating: "8.1"
  url: https://www.imdb.com/title/tt0118694
